chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -993,3 +1003,28 @@ func (b *BitcoindNotifier) GetBlock(hash *chainhash.Hash) (*wire.MsgBlock,
 
 	return b.blockCache.GetBlock(hash, b.chainConn.GetBlock)
 }
+
+// SubscribeMempoolSpent allows the caller to register a subscription to watch
+// for a spend of an outpoint in the mempool.The event will be dispatched once
+// the outpoint is spent in the mempool.
+//
+// NOTE: part of the MempoolWatcher interface.
+func (b *BitcoindNotifier) SubscribeMempoolSpent(
+	outpoint wire.OutPoint) (*chainntnfs.MempoolSpendEvent, error) {
+
+	event := b.memNotifier.SubscribeInput(outpoint)
+
+	ops := []*wire.OutPoint{&outpoint}
+
+	return event, b.chainConn.NotifySpent(ops)
```

Comment:
So you're right we don't need to load it for btcd: https://github.com/btcsuite/btcd/blob/master/rpcwebsocket.go#L903-L918